packages/table/src/headers/columnHeader.tsx

```diff
@@ -77,7 +77,7 @@ export class ColumnHeader extends React.Component<IColumnHeaderProps, {}> {
 
     public render() {
         const { grid, viewportRect, columnIndexStart, columnIndexEnd } = this.props;
-        const cells: React.ReactElement<any>[] = [];
+        const cells: Array<React.ReactElement<any>> = [];
```

Comment:
`JSX.Element[]` is fine; check out the rule logic for more specifics. types with generics are non-simple; we deem the readability of `React.ReactElement<any>[]` to be too poor